tests/bot/rules/test_links.py

```diff
@@ -0,0 +1,101 @@
+import unittest
+from typing import List, NamedTuple, Tuple
+
+from bot.rules import links
+from tests.helpers import async_test
+
+
+class FakeMessage(NamedTuple):
+    author: str
+    content: str
+
+
+class Case(NamedTuple):
+    recent_messages: List[FakeMessage]
+    relevant_messages: Tuple[FakeMessage]
+    culprit: Tuple[str]
+    total_links: int
+
+
+def msg(author: str, total_links: int) -> FakeMessage:
+    """Makes a message with `total_links` links."""
```

Comment:
Minor formatting nit; argument names should be in asterisks.
```suggestion
    """Makes a message with *total_links* links."""
```

Reply:
This is new to me, thank you! Committed in https://github.com/python-discord/bot/pull/641/commits/760f265339b1462b60ea7d5af1fd8d4476c134f0.

Reply:
args are in back ticks in all the docstrings I encountered so far so I think it'd be better to just be consistent

Reply:
it is true that backticks is the prevailing style for this in our repo, and that until we decide to change that, it's probably best to keep doing that. I've never really seen the asterisks thing, and afaik there's nothing about this in PEP257. Can I ask where you picked up that convention @aeros?

Reply:
@lemonsaurus 
> Can I ask where you picked up that convention @aeros, or what the reasoning behind it is?

We follow that convention in the CPython docs and docstrings. I don't know the specific reason behind it, but we use backticks for functions or methods, and asterisks for arg names. For example:

```
`loop.run_in_executor()`
*executor*
```

Backticks for argument names as well though is probably fine, as long as you're consistent with it.

Reply:
I'll do some poking around, to see if I can find an actual reason for it or if it's mentioned in a PEP somewhere.

Reply:
Interesting. I suppose I can see the usefulness in denoting callables one way and arguments another, what symbol is used might just be arbitrary.

Thanks for the info.